app/services/hosts.ts

```diff
@@ -48,7 +48,7 @@ export class HostsService extends Service {
   }
 
   get platform() {
-    return 'platform.streamlabs.com';
+    return 'dev-platform.streamlabs.com';
```

Comment:
We currently release our dev builds from this branch, so I can't merge this until this change is backed out.  I'm not sure what the deploy status is on the APIs this uses, but if they're not out yet we should make sure this doesn't break.

Reply:
good call, I was about to ask Runxi to deploy to platform first before merging this. On hold for now.